src/api/index.js

```diff
@@ -333,6 +340,14 @@ export function getNamespaces() {
   return get(uri).then(checkData);
 }
 
+export function getServiceAccount(name1, namespace1) {
```

Comment:
```js
export function getServiceAccount({ name, namespace }) {
  const uri =getKubeAPI('serviceaccounts', { name, namespace });
  return get(uri);
}
```

Reply:
Have tried this with the code above however it failed when we run it. 
When we console.log() the name and namespace they both were undefined 

It worked when we did it as (name, namespace) instead of (name1, namespace1)

Reply:
to use the code I provided you'd also need to update where it's called to pass in the params as an object

Reply:
done 😸